modules/sdk/project-templates/project-templates-theme/src/main/resources/archetype-resources/build.gradle

```diff
@@ -14,11 +14,11 @@ apply plugin: "com.liferay.portal.tools.theme.builder"
 
 dependencies {
 #if (${liferayVersion.startsWith("7.0")})
-	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.0.0"
+	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.4"
 #elseif (${liferayVersion.startsWith("7.1")} || ${liferayVersion.startsWith("7.2")} || ${liferayVersion.startsWith("7.3")})
-	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.0.3"
+	cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.4"
 #else
-    cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.3"
+    cssBuilder group: "com.liferay", name: "com.liferay.css.builder", version: "3.1.4"
```

Comment:
@Seiphon I think we decided to manually upgrade these, right? Like we need to maintain the different versions across releases?

Reply:
Oh, sorry, just forgot it, I will resent it. About the sf failure, it's not about my fix, I have asked Alan Huang, he said it will be fixed on PoshiCore, but for the relevant failures, I have no idea about them, I will try to rerun

Reply:
The soy-builder test issue is not about my fix, it also exist on master code, I am wondering is there a team resposible for it or I should fix it in this pr?

Reply:
I guesse it may caused by updating the Soy version to 2019-03-07, the generated js code by soy is different now, here is the commit https://github.com/liferay/liferay-portal/commit/9a9862aa0ad51121d2f81ead1a9a509d7455a9e8

Reply:
I sent the fix for the soy test failures here: https://github.com/drewbrokke/liferay-portal/pull/1556